ignite/pkg/cosmosclient/consensus.go

```diff
@@ -0,0 +1,66 @@
+package cosmosclient
```

Comment:
I moved this out of the cosmosclient file so that any non-generic functionality is separated into files